src/jit/valuenum.cpp

```diff
@@ -245,6 +245,35 @@ TFp FpRem(TFp dividend, TFp divisor)
     return (TFp)fmod((double)dividend, (double)divisor);
 }
 
+template <typename TFp, typename TFpTraits>
+TFp FpDiv(TFp dividend, TFp divisor)
+{
+#ifdef _TARGET_ARM_
```

Comment:
Seems like this should kick in any time host-arch != target-arch, not just for ARM...and I'm pretty sure you should add ARM64 to this if you otherwise don't change that.